modules/transport.js

```diff
@@ -443,15 +443,15 @@ Transport.prototype.onBroadcastBlock = function(block, broadcast) {
 	}
 
 	if (block.totalAmount) {
-		block.totalAmount = block.totalAmount.toString();
+		block.totalAmount = block.totalAmount.toNumber();
 	}
 
 	if (block.totalFee) {
-		block.totalFee = block.totalFee.toString();
+		block.totalFee = block.totalFee.toNumber();
 	}
 
 	if (block.reward) {
-		block.reward = block.reward.toString();
+		block.reward = block.reward.toNumber();
 	}
```

Comment:
In the schema of 1.1.0 these attributes are still Bignum objets. 


https://github.com/LiskHQ/lisk/blob/75e81993bff1f2fa12a9934e8dca4dde54bc833f/logic/block.js#L340-L351

In schema of 1.0.0 these are still integers

https://github.com/LiskHQ/lisk/blob/77ee83c38d086e4c0153dde740929dd0d4acacdd/logic/block.js#L341-L352

So how the method `__private.broadcaster.broadcast(` knows that its broadcasting to a node running 1.0.0 or 1.1.0?